vhdbuilder/scripts/windows/automate_version_bump.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -euxo pipefail
+
+source vhdbuilder/scripts/windows/automate_helpers.sh
+
+az login --identity
+
+pr_title="VHDVersion Bumping"
+
+latest_image_version_2019=""
+latest_image_version_2022=""
+branch_name=""
+
+set +x
+github_access_token=$1
+set -x
+
+# This function finds the latest windows VHD base Image version from the command az vm image show
+find_latest_image_version() {
+    latest_image_version_2019=$(az vm image show --urn MicrosoftWindowsServer:WindowsServer:2019-Datacenter-Core-smalldisk:latest --query 'id' -o tsv | awk -F '/' '{print $NF}')
+    latest_image_version_2022=$(az vm image show --urn MicrosoftWindowsServer:WindowsServer:2022-Datacenter-Core-smalldisk:latest --query 'id' -o tsv | awk -F '/' '{print $NF}')
```

Comment:
It assumes that ws2022 gen1 and gen2 always have the same image version.

Reply:
fixed.